src/test/TestGhoCcipSteward.t.sol

```diff
@@ -240,16 +240,186 @@ contract TestGhoCcipSteward is TestGhoBase {
       rate: 0
     });
     vm.prank(RISK_COUNCIL);
-    vm.expectRevert();
+    vm.expectRevert(
```

Comment:
add a short comment about why it reverts please

Reply:
added - [29ccd61](https://github.com/aave/gho-core/pull/427/commits/29ccd61119b52ee02d5106d7f9e43a6fe3ec84ff)